docs/backups.rst

```diff
@@ -0,0 +1,90 @@
+ServiceInfo backups
+===================
+
+
+Getting a backup dump and restoring it locally
+----------------------------------------------
+
+Steps you can do ahead of time:
+
+* Get access to the Caktus backup server (open a tech support request).
+
+When you need to restore a backup:
+
+* Make sure you are in your ServiceInfo project directory, and not in the ServiceInfo-IRCDeploy
+  project directory::
+
+    $ git config --get remote.origin.url
+    git@github.com:theirc/serviceinfo.git
+
+* List the files in the latest backup directory and find the most recent::
+
+    $ backup_path=/mnt/rsnapshot/serviceinfo-prod/hourly.0/home/caktus-backup
+    $ backup_filename=`ssh caktus-backup@backup.caktus.lan ls $backup_path | tail -1`
+
+* Copy that file to your local directory and MAKE SURE it's named ``service_info.sql.bz2``. That's
+  not important if you're only doing a local restore, but if you're restoring to staging, our Fabric
+  scripts look for that particular filename.::
+
+    $ scp caktus-backup@backup.caktus.lan:${backup_path}/${backup_filename} service_info.sql.bz2
+
+It's about 3.7 MB as of Sept 2017.
+
+* Decompress the file::
+
+    $ bunzip2 service_info.sql.bz2
+
+Now the file is in ``service_info.sql`` (in SQL text format).
+
+Drop your existing local database and restore from the backup::
+
+    $ dropdb service_info
+    $ createdb --template=template0 service_info
+    $ psql service_info < service_info.sql
+
+There will be a bunch of errors related to the fact that the production dump has roles which your
+local DB doesn't have. It's OK to ignore them.
+
+Migrate and update the CMS 'site' so that your local server uses production content::
+
+    $ workon serviceinfo
+    $ ./manage.py migrate --noinput
+    $ ./manage.py change_cms_site --from=serviceinfo.rescue.org --to=localhost:8000
```

Comment:
Can we add a note about something I ran into? The value of the ``--to=`` option has to be a site already defined in the Django admin. When I tried with ``localhost:8020``, it failed until I managed to get into the admin and change the existing ``localhost:8000`` entry to ``localhost:8020``.

Reply:
Will do.